server/sonar-server/src/main/java/org/sonar/server/platform/db/migration/AutoDbMigration.java

```diff
@@ -77,32 +80,63 @@ void installH2() {
   @VisibleForTesting
   protected void createH2Schema(Connection connection, String dialectId) {
     DdlUtils.createSchema(connection, dialectId, false);
+    populateSchemaMigration(connection, migrationSteps.getMaxMigrationNumber());
     hackFixForProjectMeasureTreeQueries(connection);
   }
 
+  private void populateSchemaMigration(Connection connection, long maxMigrationNumber) {
+    try (PreparedStatement preparedStatement = connection.prepareStatement("INSERT INTO SCHEMA_MIGRATIONS(VERSION) VALUES (?)")) {
+      batchExecute(
+        0, maxMigrationNumber + 1,
+        preparedStatement, connection,
+        (statement, counter) -> statement.setString(1, String.valueOf(counter)));
+    } catch (SQLException e) {
+      throw new RuntimeException("Failed to insert rows into table SCHEMA_MIGRATIONS", e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00112)